internal/juju/sshKeys.go

```diff
@@ -128,7 +127,7 @@ func (c *sshKeysClient) DeleteSSHKey(input *DeleteSSHKeyInput) error {
 		k := returnedKeys[0].Result[0]
 		if input.User == utils.GetUserFromSSHKey(k) {
 			// This is the latest key, do not remove it
-			log.Warn().Msgf("ssh key from user %s is the last one and will not be removed", input.User)
+			c.Warnf(fmt.Sprintf("ssh key from user %s is the last one and will not be removed", input.User))
```

Comment:
Is the `fmt.Sprintf` redundant here, can we just: ?

```suggestion
			c.Warnf("ssh key from user %s is the last one and will not be removed", input.User)
```

Reply:
no, the 2nd argument to warn is a map[string]interface.